.github/dependabot.yml

```diff
@@ -6,8 +6,11 @@ updates:
     interval: daily
     time: "10:00"
   open-pull-requests-limit: 10
+
+- package-ecosystem: docker
+  directory: "/"
+  schedule:
+    interval: daily
   ignore:
-  - dependency-name: celery
-    versions:
-    - ">= 5.0.a"
-    - "< 5.1"
+    # Allow only patch updates for docker images automatically
```

Comment:
```suggestion
    # Only send PRs for patch versions of Docker images.
```

Reply:
It's unfortunate that this setting affects all Docker images at once. Some of our `Dockerfile`' s use multi-stage builds and contain multiple base Docker images (e.g. Node.js ones) not just the Python base image. And we only want this `ignore` setting for the Python base image. Is it possible to apply setting to a particular Docker image rather than to all Docker images?

Reply:
We can set it to individual packages if we want to.